src/test/java/net/pms/configuration/FormatConfigurationRegressionTest0.java

```diff
@@ -10,966 +10,699 @@
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "MIMETYPE_AUTO"+ "'", str0.equals("MIMETYPE_AUTO"));
-
   }
 
   @Test
   public void test42() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test42"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.VP7;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "vp7"+ "'", str0.equals("vp7"));
-
   }
 
   @Test
   public void test43() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test43"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.THREEGPP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "3gp"+ "'", str0.equals("3gp"));
-
   }
 
   @Test
   public void test44() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test44"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.MJPEG;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "mjpeg"+ "'", str0.equals("mjpeg"));
-
   }
 
   @Test
   public void test45() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test45"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.BMP;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "bmp"+ "'", str0.equals("bmp"));
-
   }
 
   @Test
   public void test46() throws Throwable {
-
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test46"); }
-
     java.lang.String str0 = net.pms.configuration.FormatConfiguration.TTA;
 
     // Regression assertion (captures the current behavior of the code)
     org.junit.Assert.assertTrue("'" + str0 + "' != '" + "tta"+ "'", str0.equals("tta"));
-
   }
 
   @Test
   public void test47() throws Throwable {
+    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC_LC;
 
-    if (debug) { System.out.format("%n%s%n","RegressionTest0.test47"); }
+    // Regression assertion (captures the current behavior of the code)
+    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac-lc"+ "'", str0.equals("aac-lc"));
+  }
 
-    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC;
+  @Test
+  public void testAACMain() throws Throwable {
+    java.lang.String str0 = net.pms.configuration.FormatConfiguration.AAC_MAIN;
 
     // Regression assertion (captures the current behavior of the code)
-    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac"+ "'", str0.equals("aac"));
-
+    org.junit.Assert.assertTrue("'" + str0 + "' != '" + "aac-main"+ "'", str0.equals("aac-main"));
```

Comment:
@Sami32 
> Are you searching trouble ? Or it's just an other way to say that you're always right and when not we need to report to the first article ?

I don't understand this except that it's something about me "always having to be right".

> Maybe because we call it LibMediaInfoParser.java for a reason ?

I can't see that I've used ```aac-main``` in ```LibMediaInfoParser``` at all.

> But if you prefer i can refer to GSpot lookup tables ?

I don't know what I've done this time, but it's obvious that you want to "get back at me" for something. It would be easier if you'd tell me what that is.

> You seem take it personnally the most little comment that is not a typo, and still it's limit... That's said, that could be only my incorrect perception.

This I also get that is some kind of "revenge", but I don't understand what you mean. In particular ```and still it's limit...``` is Greek to me.